backend/app/src/pong/pong.service.ts

```diff
@@ -464,6 +469,15 @@ export class PongService {
     this.gamesByGameId.set(gameId, gameManager)
     this.gamesByUser.set(leftUser.uid, { manager: gameManager, side: 'left' })
     this.gamesByUser.set(rightUser.uid, { manager: gameManager, side: 'right' })
+
+    let user = await this.userService.findOneByUid(leftUser.uid)
+    user.status = Status.GAME
+    this.userService.update(user)
+
+    user = await this.userService.findOneByUid(rightUser.uid)
+    user.status = Status.GAME
+    this.userService.update(user)
```

Comment:
```suggestion
    const updateUser = async (uid: number, userFn: (user: User) => void) => {
      const user = await this.userService.findOneByUid(uid)
      userFn(user)
      this.userService.update(user)
    }
    const updateToGameStatus
      = async (uid: number) => await updateUser(uid, (user: User) => { user.status = Status.GAME })
      
    updateToGameStatus(leftUser.uid)
    updateToGameStatus(rightUser.uid)
```
이런식으로 `updateUser` 함수같은 헬퍼 함수가 있어도 좋을 것 같네요